enterprise/models.py

```diff
@@ -711,15 +712,16 @@ def notify_enrolled_learners(self, catalog_api_user, course_id, users, admin_enr
         with mail.get_connection() as email_conn:
             for user in users:
                 login_or_register = 'register' if isinstance(user, PendingEnterpriseCustomerUser) else 'login'
-                destination_url = destination_url.format(login_or_register=login_or_register)
+                enterprise_url = enterprise_url.format(login_or_register=login_or_register)
```

Comment:
Re: the Start My Course link: For me two questions are pertinent:
learner portal on/off:
pending vs pre-existing edx user
The learners in the bulk enroll set can be in either of these boats?
For ref: the enterprise-catalog has a piece of logic to determine enrollment_url which is used in Integrated channels for example. https://github.com/edx/enterprise-catalog/blob/3ad56d1bbad03c86123f74f7e67fcc6275a40238/enterprise_catalog/apps/catalog/models.py See get_content_enrollment_url . This one has logic to check learner portal on/off we can model from? But these urls are for enrollment. Hence I am unsure about that second question from above. Maybe Titans folks have more context too? (edited)